client/stylesheets/app.scss

```diff
@@ -1,20 +1,12 @@
-.app-content-fullscreen {
+.connection-status {
   position: fixed;
   top: 50px;
-  bottom: 0px;
   left: 0px;
   right: 0px;
+  z-index: 10000;
```

Comment:
This was specifically chosen to be greater than bootstrap's modals' z-indices, which are in the low 1000s.  (Might be worth a comment to that effect.)

Reply:
Added a comment